elections.html

```diff
@@ -6,7 +6,7 @@
 <h1>Board Elections</h1>
 <div class="blog-post clearfix">
   <p><em>Want to be part of the Computer Science Club board?</em></p>
-  <p>The TRU Computer Science Club board is a group of 10 members that run the club's day-to-day operations, organize events, manage the support lab, and work with the TRU faculty.</p>
+  <p>The TRU Computer Science Club board is a group that run the club's day-to-day operations, organize events, manage the support lab, and work with the TRU faculty.</p>
```

Comment:
Should be "is the" not "is a" or it breaks the entire sentence structure

Reply:
Actually the sentence is just broken. Could change it to be be "The TRUSU Computer Science Club board is the student group that run the club's day-to-day operations, organizes events, manages the support lab, and works with TRU faculty for the betterment of Computing Science at TRU."

or something?

Reply:
I have replaced the text as suggested.